### ISSUE TITLE
[FeatReq] autosave/ and screenshots/ should be organized diffrently

### ISSUE THREAD

**Issue by lavenderdotpet** [andrei-drexler/ironwail#273]
autosave/ should either be saved in as a .autosave/ [hidden folder] 

or the way i would prefer in /home/user/.local/share/ironwail/ 
screenshots/ should either do the same or put it in the dir the engine is running from 
and for windows u could go the route of putting stuff in the Documents\ folder 
its just annoying to keep having to delete the auto saves folder from my mod dir

**Comment by 4LT** [comment]
I'm not fond of creating more hidden files, but I do like the idea of using a subdirectory of $HOME/.local/share ala the [XDG proposal](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) instead of polluting the mod directory.

In the meantime, there is a [script I wrote that strips out extraneous files](https://github.com/4LT/qumopa/blob/master/qumopa.pyw)

**Comment by MissLavender-LQ** [comment]
yeah i prefer the freedesktop specifications setup i just said the top one if they wanted to do the bare minimum

id also suggest addons.json and addons.url.dat should also go into ./local/share/ironwail
![image](https://github.com/andrei-drexler/ironwail/assets/51702517/072c409b-f64d-4109-8ebe-424cbe533c23)

**Comment by tunbridgep** [comment]
honestly, I would love a `-userdir` command-line argument to specify the folder where autosaves, screenshots etc go.

**Comment by paperweightorder** [comment]
sorry for bump, but a -userdir argument would be really nice. or just move .ironwail to .config or .local/share
